tests/integration/test_envelope.py

```diff
@@ -597,7 +597,7 @@ def get_project_config():
 
         res = original_endpoint().get_json()
         if not include_global:
-            res.pop("global")
+            res["global"] = None
```

Comment:
What's the reason for this change?

This means `global` will be a key with a `None` value. The previous implementation, and Sentry's behavior, is not to include `global`. As a result, this test emulates a different behavior from what Sentry does.